package.json

```diff
@@ -18,15 +18,10 @@
   "homepage": "https://github.com/AleshaOleg/postcss-sass",
   "dependencies": {
     "gonzales-pe": "^4.3.0",
-    "postcss": "^7.0.31"
+    "postcss": "^8.0.9"
```

Comment:
shouldn't this become a peer dependency to follow the postcss recommendation (to avoid having multiple postcss versions in the tree, which causes problems) ?

Reply:
I was actually wondering in the case of a parser if the rule is the same ...
The migration's recommendation was about plugins.

But thinking about it now, the distinction might not be important

Reply:
one of the reason was to avoid mixing AST versions, which definitely applies to parsers too.

Reply:
Hmm, true

I made the change :)